ex2_header_css.css

```diff
@@ -0,0 +1,40 @@
+.header {
+	width: 100%;
+	height: 60px;
+	border-bottom: 1px solid;
+}
+
+.logintitle {
+	background-color: #3090C7;
+	padding: 3px;
+	text-align: center;
+	color: white;
+	box-shadow: 0px 5px 10px grey;
```

Comment:
No difference for 0vh and 0 rem because they are relative to the font size of the parent element. Difference is for 0%, the shadow will disappear.